src/dap/flatSessionConnection.ts

```diff
@@ -52,11 +55,14 @@ export class ChildConnection extends DapConnection {
   }
 
   _send(msg: Message) {
-    msg.sessionId = this.sessionId;
+    if (this.sessionId !== '') {
```

Comment:
Good point. That's the way I had it initially (sessionId undefined for the top level connection), but I didn't want to also make `IDebugSessionLike`'s `id` be possibly undefined since we use that as the key for the session map.

Should be able to use undefined for the connection but still pass the `IDebugSessionLike` id as "root" or something.